### DISCUSSION TITLE
vsomeip ASIL rating?

### DISCUSSION THREAD

**Discussion by calincirstea1** [COVESA/vsomeip#631] [Q&A] [answered]
Hello,
I am interested in the ASIL rating of the vsomeip?
Can you please let me know what is the ISO 26262 compliance status?
Manny thanks.

**Comment by goncaloalmeida** [comment, marked as answer]
@duartenfonseca vsomeip does not have an ASIL (Automotive Safety Integrity Level) rating as per ISO 26262 standards. Right?
The project does not involve any safety-related aspects that would require an ASIL rating.

cc @fcmonteiro @anaritarodrigues

**Reply by duartenfonseca** [reply]
@goncaloalmeida @calincirstea1 we do not have ASIL rating
